101-fast-bonsai/azuredeploy.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "defaultValue": "my-first-bonsai",
+      "metadata": {
+        "description": "The name of the Bonsai suite instance."
+      }
+    },
+    "location": {
+      "type": "string",
+      "allowedValues": [
+        "West US",
+        "West US 2"
+      ],
+      "metadata": {
+        "description": "The Azure location of the Bonsai suite instance. (For instant, only available in West US and West US 2.)"
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.AutonomousSystems/workspaces",
+      "apiVersion": "2020-05-01-preview",
+      "name": "[parameters('name')]",
+      "location": "[parameters('location')]",
+      "identity": {
+        "type": "systemAssigned"
```

Comment:
how is this used?

Reply:
I added a description of Bonsai in the README. Should I add the parameters also in the README.md?

Reply:
I'm referring the the managed identity - how is that used?  It's never assigned perms to anything...

Reply:
Indeed, not used (and removed).